roboteam_ai/test/ApplicationManagerTest.cpp

```diff
@@ -62,7 +63,16 @@ TEST(ApplicationManagerTest, it_handles_ROS_data) {
     rate.sleep();
     EXPECT_NE(app.strategy->getStatus(), bt::Node::Status::Waiting);
 
+    app.notifyTreeStatus(bt::Node::Status::Waiting);
+    app.notifyTreeStatus(bt::Node::Status::Running);
+    app.notifyTreeStatus(bt::Node::Status::Success);
+    EXPECT_EQ(BTFactory::getCurrentTree(), "haltStrategy");
```

Comment:
This might fail later on since halt strategy will probably wont be the default start?